app/library/ng2c3.component.ts

```diff
@@ -74,7 +75,7 @@ export class Ng2C3 {
   private ngOnChanges( changes: { [propertyName: string]: SimpleChange } ): void {
     try {
       this.__render( this.data );
-    } catch(err : Error) {
+    } catch(err) {
```

Comment:
It is augggested not to use any type for error catching ! Hence removed intentionally !